daemon/control.c

```diff
@@ -764,6 +765,9 @@ control_handle_container_stop(container_t *container, ContainerStartParams *star
 		res = -1;
 		control_send_message(CONTROL_RESPONSE_CONTAINER_STOP_PASSWD_WRONG, fd);
 	} else {
+		// TODO if the modules cannot be stopped successfully, the container is killed. The return
+		// value in this case is CONTAINER_ERROR, even if the container was killed. This is
+		// ignored atm and just STOP_OK is returned. How should we treat this?
```

Comment:
this comment was intentionally delete by the last commit.
(merge error!? plz fix)

Reply:
Fixed